src/jgikbase/idmapping/storage/mongo/id_mapping_mongo_storage.py

```diff
@@ -231,7 +235,30 @@ def get_namespace(self, namespace_id: NamespaceID) -> Namespace:
             nsdoc[_FLD_PUB_MAP],
             self._to_user_set(nsdoc[_FLD_USERS]))
 
-    def _to_user_set(self, userdocs):
-        # TODO implement when add / remove user are implemented
-        userdocs.clear()
-        return set()
+    def _to_user_set(self, userdocs) -> Set[User]:
+        return {User(AuthsourceID(u[_FLD_AUTHSOURCE]), u[_FLD_NAME]) for u in userdocs}
+
+    def add_user_to_namespace(self, namespace_id: NamespaceID, admin_user: User) -> None:
+        self._modify_namespace_users(True, namespace_id, admin_user)
```

Comment:
you might keyword the `add` parameter just so it's clearer what this arg is doing.

Reply:
Added a type and some docs locally.